<h2>Understanding the CIViC Data Model</h2>
<p>
At the heart of CIViC is the clinical evidence statement. The clinical evidence statement is a piece of information that has been manually curated from trustable medical literature about a variant or genomic 'event' that has implications in cancer diagnosis, prognosis, or predictive response to therapy. For example, “Patients with BRAF V600 mutations respond well to the drug dabrafenib”. A variant may be a single nucleotide substitution, a small insertion or deletion, a chromosomal rearrangement, an RNA gene fusion, an RNA expression pattern (e.g. over-expression), and others. Each clinical evidence statement corresponds to a single citable publication.<br><br>

The following tables define the minimal components and definitions required for a CIViC evidence statement. Once clinical evidence statements for a variant have been curated, a summary is manually generated describing the variant's significance in cancer, along with current treatment methods. A gene level summary then synthesizes the variant summaries, and adds important gene-level information. Each variant can belong to one or more variant groups, these act to combine functionally similar variants into a single entity.  For example, 'FGFR Fusions' is a group that contains several gene fusions where FGFR2 (or FGFR3) is involved in a fusion with various 3' partner genes, and 'Imatinib Resistance Mutations' is a group of variants that confer resistance to imatinib treatment. 

<h3>Minimum Requirements for Clinical Evidence Statements</h3>

 <table class="table table-striped table-condensed table-responsive table-bordered">
  <col width="167.5">
 <col width="800">
 <col width="500">
  <tr>
    <th>Data Type</th>
    <th>Description</th>
    <th>Example</th>
  </tr>
  <tr>
    <td>Gene</td>
    <td>Gene implicated (Entrez gene ID)</td>
    <td>ESR1 (Entrez ID: 2099)</td>
  </tr>
  <tr>
    <td>Variant</td>
    <td>Event (Single nucleotide variant, Insertion/deletion, Gene fusion, Copy number variant, etc.) implicated</td>
    <td>Y537S</td>
  </tr>
  <tr>
    <td>Disease</td>
    <td>Specific disease or disease subtype that is associated with this event and its clinical implication (Disease Ontology ID)</td>
    <td>Estrogen-receptor positive breast cancer (DOID: 0060075)</td>
  </tr>
  <tr>
    <td>Evidence type</td>
    <td>Category of clinical action implicated by event</td>
    <td>Predictive - The variant is predictive of sensitivity or resistance to a therapeutic.</td>
  </tr>
  <tr>
    <td>Evidence level</td>
    <td>The type of experiment from which the evidence is curated. From inferential association, to proven association in clinical medicine.</td>
    <td>Level B - Clinical Evidence</td>
  </tr>
  <tr>
    <td>Evidence direction</td>
    <td>A indicator of whether the evidence statement supports or refutes the clinical significance of an event</td>
    <td>Supports - the evidence supports the clinical significance</td>
  </tr>
  <tr>
    <td>Drug</td>
    <td>For predictive evidence, indicates the therapy for which sensitivity or resistance is indicated (With PubChem ID if applicable)</td>
    <td>Hormone therapy (PCID: N/A)</td>
  </tr>
  <tr>
    <td>Clinical significance</td>
    <td>Positive or negative association with treatment or diagnostic/prognostic end point</td>
    <td>Negative - mutation is associated with resistance to therapy</td>
  </tr>
  <tr>
    <td>Text summary</td>
    <td>Human readable interpretation. Free-form text summary of this event’s effect on cancer and potential clinical interpretations. This interpretation is the synthesis of all other information about an event and its relevance to clinical action and should be the living product of active discussion</td>
    <td>Studies suggest that ligand-binding-domain ESR1 mutants mediate clinical resistance to hormonal therapy and suggest that more potent estrogen receptor antagonists may be of substantial therapeutic benefit</td>
  </tr>
  <tr>
    <td>Citation</td>
    <td>Literature where the event was described/explored (PubMed ID)</td>
    <td>PMID: 24185512</td>
  </tr>
   <tr>
    <td>Trust Rating</td>
    <td>A rating on a 5-star scale, portraying the curators trust in the experiments from which the evidence is curated.</td>
    <td>5-stars - Strong, well supported evidence from a lab or journal with respected academic standing. Experiments are well controlled, and results are clean and reproducible across multiple replicates.</td>
  </tr>
</table>

<h3>Breaking Down the Evidence Statement</h3>
<p>
When curating evidence statements from published sources, to discern whether a variant has a "clinical interpretation", we use the data model below. Evidence statements can either be predictive of drug response, be correlated with prognostic outcome, or contribute to disease diagnosis. If an evidence item can not be placed in one of these categories, it likely lies outside of the scope of CIViC. However, published statements about a gene or variant that cannot be placed within these categories can still be added to the free-form clinical summary and variant summary.  

 <table class="table table-striped table-condensed table-responsive table-bordered">
 <col width="150">
 <col width="250">
 <col width="400">
 <col width="920">
  <tr>
    <th>Evidence type</th>
    <th>Evidence direction</th>
    <th>Clinical significance</th>
    <th>Example</th>
  </tr>
  <tr>
    <td rowspan=5><br><br><br><br><br><b>Predictive:</b> <br><br> <i>Evidence pertaining to a variant’s effect on therapeutic response</i></td>
    <td rowspan=2>Supports: <br><br>The experiment or study supports this variant’s response to a drug</td>
    <td>Sensitivity: <br>Variant is associated with positive response to treatment (e.g. sensitivity to drug)</td>
    <td>Breast cancer cell lines with a PIK3CA H1047R mutation showed increased sensitivity to CH5132799 compared to cells with wild-type PIK3CA gene</td>
  </tr>
  <tr>
    <td>Resistance or Non-Response: <br>Variant is associated with negative treatment response (e.g. resistance to drug)</td>
    <td>MAP2K1 P124S confers increased resistance to inhibition by AZD6244 inhibition by about 5-fold</td>
  </tr>
  <tr>
    <td rowspan=3>Does not support: <br><br>The experiment or study does not support, or was inconclusive of an interaction between the variant and a drug</td>
    <td>Sensitivity: <br>Variant is associated with positive response to treatment (e.g. sensitivity to drug)</td>
    <td>BRAF V600E mutation does not predict response in patients treated with dacarbazine or temozolomide</td>
  </tr>
  <tr>
    <td>N/A: <br>Variant does not inform clinical action</td>
    <td>There is no statistical difference in progression free survival between lung cancer patients treated with gefitinib or erlotinib in the EGFR L858R setting.</td>
  </tr>
  <tr>
    <td>Resistance or Non-Response: <br>Variant is associated with negative treatment response (e.g. resistance to drug)</td>
    <td> Daunorubicin treatment resulted in similar overall survival and disease free survival in de novo AML patients with DNMT3A R882 mutation compared to those who do not harbor this mutation.</td>
  </tr>

  <tr>
    <td rowspan=4><br><br><br><b>Diagnostic:</b> <br><br> <i>Evidence pertaining to a variant’s impact on patient diagnosis</i></td>
    <td rowspan=2>Supports: <br><br>The experiment or study supports variant’s impact on the diagnosis of disease or subtype</td>
    <td>Positive: <br>Variant is associated with diagnosis of disease or subtype</td>
    <td>BRAF V600E is shown to be associated with the tall-cell variant of papillary thyroid carcinoma (PTC)</td>
  </tr>
  <tr>
    <td>Negative: <br>Variant is associated with the lack of diagnosis of disease or subtype</td>
    <td>JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).</td>
  </tr>
  <tr>
    <td rowspan=2>Does not support: <br><br>The experiment or study does not support the variant’s impact on diagnosis of disease or subtype</td>
    <td>Positive: <br>Variant is associated with diagnosis of disease or subtype</td>
    <td>Frequency of NPM1 mutations was not different in normal karyotype acute myeloid leukemia patients with CEPBA, NRAS or KIT mutations. </td>
  </tr>
  <tr>
    <td>Negative: <br>Variant is associated with the lack of diagnosis of disease or subtype</td>
    <td>Study 1 found that, contrary to what was previously believed, mutation X does NOT in fact show a correlation with disease subtype A</td>
  </tr>

  <tr>
    <td rowspan=5><br><br><br><b>Prognostic:</b> <br><br> <i>Evidence pertaining to a variant’s impact on disease progression, severity, or patient survival</i></td>
    <td rowspan=2>Supports: <br><br>The experiment or study supports a variant’s impact on prognostic outcome</td>
    <td>Good Outcome: <br>Variant is associated with a better overall patient outcome</td>
    <td>In AML, patients with IDH2 R140K mutation have improved overall survival compared to those with wild-type IDH2</td>
  </tr>
  <tr>
    <td>Poor Outcome: <br>Variant is associated with a worse overall patient outcome</td>
    <td>Meta-analysis of studies involving cytogenetically normal, younger patients showed increased overall and relapse-free survival for patients with NPM1 mutations</td>
  </tr>
  <tr>
    <td rowspan=3>Does not support: <br><br>The experiment or study does not support a prognostic association between variant and outcome</td>
    <td>Good Outcome: <br> Variant is associated with a better overall patient outcome</td>
    <td>Mutation X was not shown to be associated with improved overall survival</td>
  </tr>
  <tr>
    <td>N/A: <br>Variant does not inform clinical action</td>
    <td>Size of the FLT3-ITD mutant duplication had no impact on overall survival or relapse rate</td>
  </tr>
  <tr>
    <td>Poor Outcome: <br>Variant is associated with a worse overall patient outcome</td>
    <td>Unlike other studies that suggest a poorer outcome, BRAF mutation in this study was not correlated with poorer prognosis in papillary thyroid cancer.</td>
  </tr>
</table>


<h3>Evidence Levels: Overview</h3>
<p>
Each evidence statement is the result of an experiment, trial or study in published literature. It is important to capture the nature of these experiments in the evidence entry. Evidence levels allow for the subject of an evidence item to be presented in a simple, standardized fashion. 

 <table class="table table-striped table-condensed table-responsive table-bordered">
 <col width="150">
 <col width="300">
 <col width="920">
  <tr>
    <th>Level of Evidence</th>
    <th>Definition</th>
    <th>Example</th>
  </tr>
  <tr>
    <td>A - Validated association</td>
    <td>Proven/consensus association in human medicine</td>
    <td>A meta-analysis of clinical studies showed that harboring a BRAF V600E mutation predicts worse prognosis in patients with colorectal cancer</td>
  </tr>
  <tr>
    <td>B - Clinical evidence</td>
    <td>Clinical trial or other primary patient data supports association</td>
    <td>In non-small-cell lung cancer patients with EGFR T790M and other activating mutations, their progression-free survival is shorter than those who do not have T790M mutations</td>
  </tr>
  <tr>
    <td>C - Preclinical evidence</td>
    <td>In vivo or in vitro models support association</td>
    <td>Experiments showed that AG1296 is effective in triggering apoptosis in cells with the FLT3 internal tandem repeat</td>
  </tr>
  <tr>
    <td>D - Inferential association</td>
    <td>Indirect evidence</td>
    <td>Glioma cells harboring IDH1 mutation may be more susceptible to chemotherapy or radiotherapy due to their reduced ability to respond to oxidative stress</td>
  </tr>
</table>

<h3>Trust Ratings Explained</h3>
<p>
In order to quickly discern how much trust curators and users have in a single evidence statement, a five star trust ratings system is used. Each evidence item is given a rating, from 1 to 5 stars, based on the quality of the source from which the evidence is curated. These ratings depend on a number of factors, including journal impact, control quality, orthogonal validation, and reproducibility. 

 <table class="table table-striped table-responsive table-bordered">
 <col width="45">
 <col width="600">
  <tr>
    <th>Trust Rating</th>
    <th>Definition</th>
  </tr>
  <tr>
    <td>1 star</td>
    <td>Claim is not supported well by experimental evidence. Results are not reproducible, or have very small sample size. No follow-up is done to validate novel claims.</td>
  </tr>
  <tr>
    <td>2 stars</td>
    <td>Evidence is not well supported by experimental data, and little follow-up data is available. Publication is from a journal with low academic impact. Experiments may lack proper controls, have small sample size, or are not statistically convincing.</td>
  </tr>
  <tr>
    <td>3 stars</td>
    <td>Evidence is convincing, but not supported by a breadth of experiments. May be smaller scale projects, or novel results without many follow-up experiments. Discrepancies from expected results are explained and not concerning.</td>
  </tr>
  <tr>
    <td>4 stars</td>
    <td>Strong, well supported evidence. Experiments are well controlled, and results are convincing. Any discrepancies from expected results are well-explained and not concerning.</td>
  </tr>
  <tr>
    <td>5 stars</td>
    <td>Strong, well supported evidence from a lab or journal with respected academic standing. Experiments are well controlled, and results are clean and reproducible across multiple replicates. Evidence confirmed using separate methods.</td>
  </tr>
</table>

<!--
{{ loadedMsg }}
-->
